src/init.cpp

```diff
@@ -1518,33 +1522,23 @@ bool AppInitMain(InitInterfaces& interfaces)
                 // At this point we're either in reindex or we've loaded a useful
                 // block tree into BlockIndex()!
 
-                pcoinsdbview.reset(new CCoinsViewDB(nCoinDBCache, false, fReset || fReindexChainState));
```

Comment:
In commit "refactor: have CCoins* data managed under CChainState" (eed4e62ad6b162ff0d3e2b87eb8762a689678bca)

Could you specify in the commit message whether this is changing initialization order in any significant way? It seems like utxo database is being loaded a lot earlier now.